Fix: add sop field button (#3248)
miso-lims/miso-lims: changes/fix_addSopFieldButton.md

> Add button for sop field were not showing in create sop before saving sop. 

Reviewer: The "Remove" button wasn't appearing either, and SOP should be capitalized

```suggestion
Buttons for managing SOP fields were not showing on the Create SOP page
```
